src/js/jsx/sections/transform/Size.jsx

```diff
@@ -78,6 +139,64 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Saves the current width of selected layers on scrub start
+         *
+         * @private
+         */
+        _handleWScrubBegin: function () {
+            this._initialScrubW = collection.uniformValue(this.state.widths);
+        },
+
+        /**
+         * Update the current width of the selected layers based on change from initial scrub position
+         *
+         * @private
+         * @param {number} deltaX
+         */
+        _handleWScrub: function (deltaX) {
+            if (_.isNumber(this._initialScrubW)) {
```

Comment:
Yes, I agree. As I move things to state, instead of computing state on render, this will be easier to do.